### DISCUSSION TITLE
coding/

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#21] [General]
# coding/

编程 写在前面 编程领域，本人比较喜欢 ToC 软件开发[1]，实用至上，自由主义，热爱开源。 我的思考 语言有关 我的编程语言学习属于多而不精的类型。多语言学习者能在短时间内掌握一门新的语言，但这也意味着基础不够扎实，缺乏经验等。因此需要多做项目以降低劣势。 多语言只是手段，目的是为了开拓视野，总结出不同语言中的共性[2]。编程学习不能仅仅“见树”，...

https://absx.pages.dev/coding/

**Comment by s-chance** [comment]
https://www.criwits.top/missing/ 也不错，可能偏小白一点
